public/javascripts/SVLabel/css/svl.css

```diff
@@ -8,10 +8,10 @@
 text {
     visibility: hidden;
 }
-
+/*
```

Comment:
The sounds and stats are given the class "visible" which overrides the "hidden" visibility of the parent container. At least for the sound button, it uses the visible class because the mute button uses the hidden class (they switch off).